nimare/__init__.py

```diff
@@ -41,34 +40,3 @@
     ]
 
 del get_versions
-
-
-def _py367_deprecation_warning():
```

Comment:
This section of code still seems useful, but maybe we add back in later...

otherwise, I could see us tracking the minimum python version that we support and when python is ~3 months away from dropping support from that particular version, this code would become active again. This would also require changes to the developer notes for release.

This could be another issue/PR that is done later.